pkg/test/project.csproj.template

```diff
@@ -11,5 +11,5 @@
     <PackageReference Include="{PackageId}" Version="{PackageVersion}" />
   </ItemGroup>
 
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), packageTest.targets))\packageTest.targets" />
+  <Import Project="$(SupportFilesDir)\packageTest.targets" />
```

Comment:
nit: Consider having a default so that local builds don't require a global property to be passed in order to run an individual test.

Reply:
Test.msbuild will be passing that property and it has a default in it: https://github.com/dotnet/corefx/pull/34764/files/4413acd0e63d5e7adcff14f81044b708e0b2939d#diff-3e8fa4218f365c2255f5e225c7a7bfe9R9 -- do you think it is a common scenario that people run the tests manually directly by calling project.csproj directly? Or the common scenario is always doing it through testPackages.proj?

But the default wouldn't hurt anything, it should just point to, 2 directories above.

Reply:
nothing here is a common scenario.  it's probably only a handful of people running these manually.  I don't feel strongly about it, but it was something I had in mind when I did this stuff originally.  Since we generate everything I felt like it was possible / better to not have to specify params.

Reply:
Ok, I guess it doesn't hurt adding a default value, so I added it into the project template. PTAL.